container-toolkit/release-notes.md

```diff
@@ -10,6 +10,69 @@ This document describes the new features, improvements, fixed and known issues f
 
 ______________________________________________________________________
 
+## NVIDIA Container Toolkit 1.16.2
+
+**This release provides critical security updates and is recommended for all users.**
+
+It includes updates for:
+* [NVIDIA CVE-2024-0132](https://nvidia.custhelp.com/app/answers/detail/a_id/5582)
+* [NVIDIA CVE-2024-0133](https://nvidia.custhelp.com/app/answers/detail/a_id/5582)
+
+To view any published security bulletins for NVIDIA products, see the NVIDIA product security page (https://www.nvidia.com/en-us/security/)
+
+For more information regarding NVIDIA's security vulnerability remediation policies, see (https://www.nvidia.com/en-us/security/psirt-policies/)
```

Comment:
nit: `NIVIDIA security...`  The guideline is not to form the possessive or hyphenate NVIDIA.